cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -151,12 +168,14 @@ function UnmanagedClusterNetworkSettings(props: Partial<StepProps>) {
                             Node to host port mapping <CdsIcon shape="info-circle" size="md" status="info"></CdsIcon>
                         </label>
                         <input
-                            {...register('CLUSTER_NAME')}
+                            {...register('NODE_HOST_PORT_MAPPING')}
```

Comment:
It would be nice to define these fields (like 'NODE_HOST_PORT_MAPPING') as const, and then use them everywhere, including in the definition of the inputs to the component. That way all these places show up under usages of the string constant.